sklearn/base.py

```diff
@@ -1116,7 +1166,16 @@ def is_classifier(estimator):
     >>> is_classifier(kmeans)
     False
     """
-    return getattr(estimator, "_estimator_type", None) == "classifier"
+    # TODO(1.8): Remove this check
```

Comment:
Those deprecation needs to be announce in a specific changelog entry. There are public functions.